### PR TITLE
Add wrappers to open individual standard libraries.

### PR DESCRIPTION
**Pull request by cjcole**
For those who prefer to pick and choose which of the libraries to use.

### CODE REVIEW

**Comment by cjcole** [comment]
to_str_in_place is to match the semantics of to_numberx, to_integerx, to_bool, etc, none of which push a value onto the stack.

File: src/wrapper/state.rs [hunk 1]
```diff
@@ -1091,6 +1146,17 @@ impl State {
     }
   }
 
+  pub fn to_str_in_place(&mut self, index: Index) -> Option<&str> {
+      let mut len = 0;
```
Comment (SpaceManiac) [inline src/wrapper/state.rs]: I recommend a doc comment noting "Maps to `lua_tolstring`" and removing the current "omitted: lua_tolstring" comment. Also, minor, but there's an indentation error here.

**Comment by jcmoyer** [comment]
Thanks @cjcole for the useful PR and @SpaceManiac for reviewing.

**Comment by cjcole** [comment]
Thanks for the useful library.

**Pull request merged**